qa/workunits/fs/snaps/snaptest-name-limits.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+#
+# This tests snapshot names limits: names have to be < 240 chars
+#
+
+function cleanup ()
+{
+	rmdir d1/.snap/*
+	rm -rf d1
+}
+
+function fail ()
+{
+	echo $@
+	cleanup
+	exit 1
+}
+
+mkdir d1
+
+longname=$(printf "%.241d" 2)
+mkdir d1/.snap/$longname 2> /dev/null
+exists=$(ls d1/.snap|grep $longname)
+[ -z $exists ] || fail "Invalid snapshot exists: $exists"
```

Comment:
[ -d d1/.snap/$longname ] && fail "Invalid snapshot exists: $exists" is more readable ?

Reply:
Thank you @kotreshhr I've just pushed a new revision with your suggestion.